libvast/vast/concept/parseable/vast/json.hpp

```diff
@@ -36,20 +38,15 @@ struct json_parser : parser<json_parser> {
     auto lbrace = ws >> '{' >> ws;
     auto rbrace = ws >> '}' >> ws;
     auto delim = ws >> ',' >> ws;
-    auto null = ws >> "null"_p ->* [] { return json::null{}; };
+    auto null = ws >> "null"_p->*[] { return json::null{}; };
```

Comment:
All parser actions use whitespace around `->*`, as the DSL considers this operator as a binary operation. Please keep this consistent for now. We can discuss global refactoring as part of a separate PR, if you want to.

Reply:
> Please keep this consistent for now.

Putting whitespaces around `->*` will trigger `clang-format`. Is this really that important to you that you want me to put `// clang-format on/off` around this one line in this function?

Reply:
I think readability of the code should always trump the desire to use `clang-format`. To me, `"null"_p->*[] { ..` looks incredibly hard to understand. For all other parsers that we wrote, we have the whitespace style. Why make an exception here?